src/components/ConversionProcessingOverlay/style.css

```diff
@@ -10,53 +10,46 @@
     justify-content: center;
 }
 
-.container h2,p,h3 {
-  color: var(--dark);
+.container h2,
+.container h3 {
+    color: var(--dark);
 }
 
-.spin {
-  margin: 20px 0;
-  padding: 30px 50px 10px;
-  text-align: center;
-  border-radius: 4px;
+.title {
+    font-weight: 700;
 }
 
-.container .title-text {
-  font-weight: 700;
+.content {
+    display: flex;
+    flex-flow: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    margin: 10px 0 26px 0;
 }
 
-.container .center-text {
-  text-align: center;
-  margin-bottom: 0px;
+.go-back-button {
+    margin-top: 10px;
 }
 
-.go-back-button svg {
-  fill: var(--blue);
```

Comment:
I tested it and this wasn't necessary

Reply:
this is so strange, it's working on your branch but when i changed it before it turned the arrow black.